src/Forms/Shared/Samples/Map/SearchPortalMaps/SearchPortalMaps.xaml

```diff
@@ -0,0 +1,102 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"             
+             xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms"
+             x:Class="ArcGISRuntimeXamarin.Samples.SearchPortalMaps.SearchPortalMaps">
+  <Grid x:Name="LayoutRoot">
+    <Grid.RowDefinitions>
+      <RowDefinition/>
+      <RowDefinition Height="70"/>
+    </Grid.RowDefinitions>
+
+        <esriUI:MapView x:Name="MyMapView"
+                        Grid.Row="0"/>
+        <TableView x:Name="SearchMapsUI"
+                   Grid.Row="0"
+                   IsVisible="False"
+                   Intent="Form">
+            <TableRoot>
+                <TableSection Title="Search Public Maps">
+                    <EntryCell x:Name="SearchTextEntry"
+                               Label="Search Text:"
+                               Text="Zoning"/>
```

Comment:
A default search text string was not specified in WPF and UWP. Should the EntryCell.Text property be an empty string?